exporter/datadogexporter/metadata/host_test.go

```diff
@@ -84,7 +85,7 @@ func TestHostnameFromAttributes(t *testing.T) {
 	})
 	hostname, ok = HostnameFromAttributes(attrs)
 	assert.True(t, ok)
-	assert.Equal(t, hostname, testContainerID)
+	assert.Equal(t, hostname, testHostID)
```

Comment:
Now that `conventions.AttributeContainerID` is resolved after `conventions.AttributeHostID` and `conventions.AttributeHostName`, we should change the above test to test the `conventions.AttributeContainerID` branch, instead of changing the result to `testHostID`. Using:
```
        attrs = testutils.NewAttributeMap(map[string]string{
		conventions.AttributeContainerID: testContainerID,
	})
```

should work, and would fix the codecov warning above.

Reply:
Done!